### PR TITLE
Imrpove by minor fixes

### PR DESCRIPTION
**Pull request by yuliaveres**

### CODE REVIEW

File: Toshi/APIClients/idAPIClient.swift [hunk 1]
```diff
@@ -41,7 +41,13 @@ public class IDAPIClient: NSObject, CacheExpiryDefault {
         }
     }()
 
-    let contactUpdateQueue = DispatchQueue(label: "token.updateContactsQueue")
+    private lazy var updateOperationQueue: OperationQueue = {
+        let queue = OperationQueue()
+        queue.maxConcurrentOperationCount = 2
```
Comment (faymontage) [inline Toshi/APIClients/idAPIClient.swift]: Where does the value "2" come from? Can we give it a name?

Comment (yuliaveres) [reply Toshi/APIClients/idAPIClient.swift]: it is enough to have 1 actually

File: Toshi/Avatars/AvatarManager.swift [hunk 2]
```diff
@@ -114,25 +114,26 @@ final class AvatarManager: NSObject, CacheExpiryDefault {
     }
 
     func downloadAvatar(path: String, completion: @escaping (_ image: UIImage?) -> Void) {
+        DispatchQueue.global(qos: .userInitiated).async {
+            guard let url = URL(string: path) as URL?,
+                let teapot = self.teapot(for: url) as Teapot? else { return }
```
Comment (heumn) [inline Toshi/Avatars/AvatarManager.swift]: Shouldnt the completion block be called here? 🤔

Comment (yuliaveres) [reply Toshi/Avatars/AvatarManager.swift]: it should

**Pull request merged**